src/app/src/reducers/FiltersReducer.js

```diff
@@ -84,10 +84,10 @@ export default createReducer(
             update(state, {
                 lists: { $set: payload },
             }),
-        [resetAllFilters]: (state, payload) =>
+        [resetAllFilters]: (state, isEmbedded) =>
             update(initialState, {
                 contributors: {
-                    $set: payload
+                    $set: isEmbedded
                         ? state.contributors
```

Comment:
Makes the change that was suggested here: https://github.com/open-apparel-registry/open-apparel-registry/pull/1257/files#r581397227